src/shadowbox/server/manager_metrics.ts

```diff
@@ -15,8 +15,21 @@
 import {PrometheusClient} from '../infrastructure/prometheus_scraper';
 import {DataUsageByUser, DataUsageTimeframe} from '../model/metrics';
 
+export type TunnelTimeDimension = 'access_key' | 'country' | 'asn';
+
+interface TunnelTimeRequest {
+  hours: number;
+}
+
+interface TunnelTimeResponse {
+  location?: string;
+  asn?: number;
```

Comment:
We should return as_org as well, now that we have that. We will be using that in the UI